src/views/manage-identities/ManageIdentities.tsx

```diff
@@ -0,0 +1,403 @@
+      setUpdateTxError(undefined);
+    } catch (err: any) {
+      setUpdateTxError(err);
+      setUpdateTxResult(undefined);
+    }
+  };
+
+  const handleUpdateIdentityCountry = async (value: string | number) => {
+    try {
+      const txHash = await mutateUpdateIdentityCountry({
+        country: value as number,
+      });
+      setUpdateTxResult(txHash);
+      setUpdateTxError(undefined);
+    } catch (err: any) {
+      setUpdateTxError(err);
+      setUpdateTxResult(undefined);
+    }
+  };
+
+  const handleAgentSelect = (agent: string | number) => {
+    setSelectedAgent(agent as `0x${string}`);
+  };
+
+  const removeUserAgent = async () => {
+    try {
+      const txHash = await mutateIdentityRegistryRemoveAgent();
+      setUpdateTxResult(txHash);
+      setUpdateTxError(undefined);
+    } catch (err: any) {
+      setUpdateTxResult(undefined);
+      setUpdateTxError(err);
+    }
+  };
+
+  useEffect(() => {
+    (deployedTokens as any).map((item: any) => {
+      const tokenAddress = item["args"]?.[0];
+      tokenAddress &&
+        readTokenOwner({}, tokenAddress).then((resOwner) => {
+          resOwner[0].toString().toLowerCase() === accountEvm?.toLowerCase() &&
+            readTokenName({}, tokenAddress).then((resName) => {
+              setOwnTokens((prev) => {
+                return [
+                  ...prev.filter((itemSub) => itemSub.address !== tokenAddress),
+                  {
+                    address: tokenAddress,
+                    name: resName[0],
+                  },
+                ];
+              });
+            });
+        });
+    });
+  }, [accountEvm, deployedTokens, setOwnTokens]);
+
+  useEffect(() => {
+    if (registry) {
+      const unsubAgentsAdded: WatchContractEventReturnType =
+        watchIdentityRegistryAgentAddedEvent(
+          {
+            onLogs: (data) => {
+              setAddedAgents(((prev: any) => {
+                return [...prev, ...data.map((log: any) => log.args[0])];
+              }) as any);
+            },
+          },
+          registry as `0x${string}`,
+        );
+      return () => {
+        unsubAgentsAdded();
+      };
+    }
+  }, [registry]);
+
+  useEffect(() => {
+    if (addedAgents?.length && registry) {
+      const unsubAgentsRemoved: WatchContractEventReturnType =
+        watchIdentityRegistryAgentRemovedEvent(
+          {
+            onLogs: (data) => {
+              const agents = data.map((log: any) => log.args[0]);
+
+              if (agents.find((agent) => addedAgents.includes(agent))) {
+                setAddedAgents((prev) =>
+                  prev.filter((agent) => !agents.includes(agent)),
+                );
+              }
+            },
+          },
+          registry as `0x${string}`,
+        );
+
+      return () => {
+        unsubAgentsRemoved();
+      };
+    }
+  }, [addedAgents, registry]);
+
+  return (
+    <>
+      <Flex gap="4" pt="4" className="modify-identities-container">
+        <style>
```

Comment:
We can convert these inline styles to use chakra ui lib we have

https://v2.chakra-ui.com/docs/styled-system/responsive-styles

Reply:
we have some breakpoints in the theme
src/theme/Theme.ts

let me know, I can suggest some neat ways of using them with chakra ui